wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -314,11 +314,27 @@ extension ConversationCreationController: AddParticipantsConversationCreationDel
 
                 case .failure(let error):
                     WireLogger.conversation.error("failed to create conversation: \(String(describing: error))")
+                    showGenericErrorAlert()
                 }
             }
         }
     }
 
+    private func showGenericErrorAlert() {
+        typealias ConnectionError = L10n.Localizable.Error.Connection
```

Comment:
Connection error?

Reply:
Yes, I use the same error as for Connection

Reply:
I think it'd be better to have a dedicated error, but we could also do it later since this is time sensitive.